### PR TITLE
Update mob-name.csv

### PR DESCRIPTION
**Pull request by chrisforyst**
Quality checking names and fixing minor bugs I found

### CODE REVIEW

File: src/data/mob-name.csv [hunk 1]
```diff
@@ -290,7 +290,7 @@
 "アングータ","Anguta"
 "アイタンリ","Eitanli"
 "ジュエルジルコン","Jewel Zircon"
-"ビロンギングス","Billongings"
+"ビロンギングス","Bellongings"
```
Comment (FabulousCupcake) [inline src/data/mob-name.csv:293]: not Belongings?

Comment (chrisforyst) [reply src/data/mob-name.csv:293]: Hmmm.... yeah, Belongings sounds better.

Comment (FabulousCupcake) [reply src/data/mob-name.csv:293]:
```suggestion
"ビロンギングス","Belongings"
```

**Review by chrisforyst — COMMENTED** [review]
Okay, I think all are now correct

**Review by FabulousCupcake — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
Thank you!

**Pull request merged**